changelog/_unreleased/2020-09-17-search-for-mail-templates.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Search for mail templates
+issue: NEXT-7072
+author: Sebastian Hölscher
+author_email: s.hoelscher@shopware.com 
+author_github: @HoelShare
+---
+# Administration
+*  Added the search term to the search request, for searching mail templates and mail header footer.
```

Comment:
Please provide more specific information, for example:

* Added new optional prop `searchTerm` to `sw-mail-header-footer-list/index.js`
* Added new boolean data prop `disableRouteParams` to `sw-mail-header-footer-list/index.js`
* ...